src/repository/base.repository.js

```diff
@@ -38,6 +38,17 @@ class BaseRepository {
     return model.findOne({ where: options });
   }
 
+  /**
+   * @static
+   * @param {*} model
+   * @param {*} options
+   * @returns {object} - returns an database object
+   * @memberof BaseRepository
+   */
+  static async find(model, options) {
```

Comment:
There is already a `FindOneByField` method in the baserepository. Please remove this

Reply:
I think the baserepository module is meant to provide the team with methods that can be useful to the team as they work on the project.

As the project progresses, the team can decide on what methods and modules to retain going forward